R/read_extract_delayed_discharges.R

```diff
@@ -0,0 +1,18 @@
+#' Read Delayed Discharges extract
+#'
+#' @param year Year of BOXI extract
+#'
+#' @return zsav/rds data file for Delayed discharges
+#' @export
+#'
+read_extract_delayed_discharges <- function(year) {
+  extract_delayed_discharges <- haven::read_sav(get_dd_path(ext = "zsav")) %>%
```

Comment:
Is it sensible to hard-code a `.zsav` extension? Presumably in the future we won't be receiving the extract in this format

Reply:
The function `get_dd_path` is already hard-coded in `.rds` format, so eventually we will not need the `ext = .zsav` here. This is only specified as the delayed discharges team will be converting their script into R so until then, we will only have the `.zsav` versions available. Can you think of another way of doing this in the meantime? otherwise i will add an issue to review this when we have the `.rds` versions available from the team